api/v1beta1/common_types.go

```diff
@@ -101,6 +101,83 @@ type BarbicanComponentTemplate struct {
 	NetworkAttachments []string `json:"networkAttachments,omitempty"`
 }
 
+// BarbicanLunaTemplate - Variables specific to Thales' Luna HSMs
```

Comment:
Lets avoid using company names like "Thales".  The companies have changed a bunch!